setup.cfg

```diff
@@ -1,49 +0,0 @@
-[metadata]
-name = readthedocs
-version = 11.11.0
-license = MIT
-description = Read the Docs builds and hosts documentation
-author = Read the Docs, Inc
-author_email = dev@readthedocs.com
-url = http://readthedocs.org
-classifiers = 
-	Development Status :: 5 - Production/Stable
-	Environment :: Web Environment
-	Intended Audience :: Developers
-	License :: OSI Approved :: MIT License
-	Operating System :: OS Independent
-	Programming Language :: Python
-	Programming Language :: Python :: 2.7
-	Programming Language :: Python :: 3.4
-	Programming Language :: Python :: 3.5
-	Programming Language :: Python :: 3.6
-	Framework :: Django
-
-[options]
-packages = find:
-include_package_data = True
-zip_safe = False
-
-[tool:release]
-github_owner = readthedocs
-github_repo = readthedocs.org
-
-[bumpver]
-current_version = "11.11.0"
-version_pattern = "MAJOR.MINOR.PATCH[TAGNUM]"
-commit_message = "Bump version {old_version} -> {new_version}"
-commit = False
-tag = False
-push = False
-
-[bumpver:file_patterns]
-setup.cfg = 
-	version = {version}
```

Comment:
This pattern seems to be missing from the pyproject.toml

Reply:
We deleted `setup.cfg`!

Reply:
This match still exists in pyproject.toml, at https://github.com/readthedocs/readthedocs.org/pull/11726/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R8

This pattern needs to be added to the configuration in pyproject.toml